config.md

```diff
@@ -95,6 +95,7 @@ See links for details about [mountvol](http://ss64.com/nt/mountvol.html) and [Se
 **`process`** (object, required) configures the container process.
 
 * **`terminal`** (bool, optional) specifies whether you want a terminal attached to that process, defaults to false.
+* **`initialConsoleSize`** (box, optional) specifies the initial console size of the terminal if attached.
```

Comment:
Yes Linux has tty console size, but the `Window` in the name `consoleWindowSize` doesn't mean `Windows operate system`, it just reflect semantics from http://man7.org/linux/man-pages/man4/tty_ioctl.4.html (section `Get and set window size`).

And I don't think we need to specify `initial` for the usage of initialization, otherwise we'll call `InitialArgs`, `InitialRlimits`, etc.. Not to mention that they can also be used in non-initialization stages.